### PR TITLE
Add some rz-diff -t lines tests

### PR DESCRIPTION
**Pull request by kazarmy**
**Your checklist for this pull request**
- [X] I've read the [guidelines for contributing](https://github.com/rizinorg/rizin/blob/master/DEVELOPERS.md) to this repository
- [X] I made sure to follow the project's [coding style](https://github.com/rizinorg/rizin/blob/master/DEVELOPERS.md#code-style)
- [X] I've added tests that prove my fix is effective or that my feature works (if possible)
- [ ] I've updated the documentation and the [rizin book](https://github.com/rizinorg/book) with the relevant information (if needed)

**Detailed description**

This pr adds some `rz-diff -t lines` tests as well as do https://github.com/rizinorg/rizin/pull/1174#discussion_r642113448.

**Test plan**

All builds are green.

**Closing issues**

...

### CODE REVIEW

File: test/db/tools/correct_cmd_i.txt [hunk 1]
```diff
@@ -0,0 +1,41 @@
+[Sections]
```
Comment (wargio) [inline test/db/tools/correct_cmd_i.txt]: Please upload this here: https://github.com/rizinorg/rizin-testbins/tree/master/other/rz-diff

File: test/db/tools/incorrect_cmd_i.txt [hunk 2]
```diff
@@ -0,0 +1,42 @@
+[Sections]
```
Comment (wargio) [inline test/db/tools/incorrect_cmd_i.txt]: same as above

**Review by wargio — CHANGES_REQUESTED** [review]

File: test/db/tools/correct_cmd_i.txt (hunk 1 above)
Comment (kazarmy) [reply test/db/tools/correct_cmd_i.txt]: Ok https://github.com/rizinorg/rizin-testbins/pull/35

File: test/db/tools/incorrect_cmd_i.txt (hunk 2 above)
Comment (kazarmy) [reply test/db/tools/incorrect_cmd_i.txt]: Ok https://github.com/rizinorg/rizin-testbins/pull/35

**Comment by wargio** [comment]
that fail seems to be quite consistant

**Comment by kazarmy** [comment]
yeah even up to the missing log file ... can't see a core dump though

**Comment by wargio** [comment]
try to build it with asan and see what is wrong

**Comment by kazarmy** [comment]
both tests fail completely (no output) on openbsd ... core dump on the second (color) test

openbsd_coredump

**Comment by XVilka** [comment]
Please push again - it seems that webhooks weren't working properly in GitHub too.

**Comment by kazarmy** [comment]
I'll wait until asan goes green and then revert 5ec2f78.

**Review by wargio — APPROVED** [review]

**Pull request merged**